models/order.js

```diff
@@ -2,39 +2,34 @@ var mongoose = require('mongoose');
 var cfg = require('../config');
 var twilio = require('twilio');
 
+
 var OrderSchema = new mongoose.Schema({
-  customerName:String,
+  customerName: String,
   customerPhoneNumber: String,
-  status : { type: String, default: 'Ready' },
-  notificationStatus : { type: String, default: 'None' },
+  status: {type: String, default: 'Ready'},
+  notificationStatus: {type: String, default: 'None'},
 });
 
-OrderSchema.methods.sendSmsNotification = function (message, statusCallback , callback) {
+OrderSchema.methods.sendSmsNotification = function(message, statusCallback, callback) {
 
-  var client = new twilio.RestClient(cfg.twilioAccountSid, cfg.twilioAuthToken);
+  var client = twilio(config.accountSid, config.authToken);
```

Comment:
This won't work because the `config` variable is not defined. Which means that also some tests will fail. Surprisingly Travis is telling that the build is green, but this reveals a subtle problem, the build is not running the test suite.

Could you please use `cfg` instead of `config` or the other way around. 
One quick fix for having the tests working in the right way is to include a test script in the `package.json` file.

I'd love to hear your thoughts on this.